myapp/spec/models/task_label_spec.rb

```diff
@@ -3,5 +3,23 @@
 require 'rails_helper'
 
 RSpec.describe TaskLabel, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'association' do
+    it 'belongs to task' do
+      r = described_class.reflect_on_association(:task)
+      expect(r.macro).to eq(:belongs_to)
+    end
+
+    it 'belongs to label' do
+      r = described_class.reflect_on_association(:label)
+      expect(r.macro).to eq(:belongs_to)
+    end
+
+    it 'is successfully deleted when a parent task is deleted' do
+      task = FactoryBot.create(:task)
+      label = FactoryBot.create(:label)
```

Comment:
テスト内で使う変数は [`let`](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let) を使う方法もあるので、ご参考に！
https://qiita.com/jnchito/items/cdd9eef2ed193267c651

```suggestion
        let(:task)  { FactoryBot.create(:task) }
        let(:label) { FactoryBot.create(:label) }
        it 'is successfully deleted when a parent task is deleted' do
```